src/plugins/workspace/common/constants.ts

```diff
@@ -11,6 +11,8 @@ export const WORKSPACE_CREATE_APP_ID = 'workspace_create';
 export const WORKSPACE_LIST_APP_ID = 'workspace_list';
 export const WORKSPACE_DETAIL_APP_ID = 'workspace_detail';
 export const WORKSPACE_INITIAL_APP_ID = 'workspace_initial';
+export const ESSENTIAL_OVERVIEW_APP_ID = 'essential_overview';
+export const ANALYTICS_OVERVIEW_ALL_APP_ID = 'analytics_all_overview';
```

Comment:
```suggestion
export const ESSENTIAL_OVERVIEW_APP_ID = 'essentials_overview';
export const ANALYTICS_OVERVIEW_ALL_APP_ID = 'all_overview';
```
Can we use the default_nav_group from core to construct the id for overview pages?

Reply:
Seems analytics all overview page exists even when workspace is disabled, should we put the app_id to home page?

Reply:
`analytics all` use case only for workspace, workspace disable case, it will be home page.